packages/Webkul/API/Http/Resources/Catalog/Product.php

```diff
@@ -56,7 +56,7 @@ public function toArray($request)
                     $product->getTypeInstance()->haveSpecialPrice(),
                     core()->currency($product->getTypeInstance()->getSpecialPrice())
                 ),
-            $product->getTypeInstance()->getProductPrices(),
+            'prices'                 => $product->getTypeInstance()->getProductPrices(),
```

Comment:
Hi, I guess you only needed a `regular_price` because a special price is already coming. If you check the pattern there are two keys `special_price` and `formated_special_price`.

I encourage you to follow the consistent pattern and try to add `regular_price` and `formated_regular_price` rather than the whole array.

Reply:
Added `regular_price` and `formated_regular_price` keys instead of the whole array